README.md

```diff
@@ -90,11 +90,74 @@ profile_lustre::tuning::params:
 
 Recommendations and/or defaults TBD.
 
+### Telegraf Monitoring
 
-## Dependencies
+Can use telegraf to monitor lustre client health
+
+To enable/disable set:
+```yaml
+profile_lustre::service::lnet_service_enabled: true
+```
```

Comment:
I'm confused what the lustre lnet service being enabled has to do with monitoring.

Are you just saying it doesn't do any monitoring if the lnet service is not enabled? If so, perhaps this can be worded better.

Or is this a typo and should instead be:
```
profile_lustre::telegraf::lustre_client_health::enabled: true
```

Reply:
Ha yeah that's a typo, supposed to be `profile_lustre::telegraf::lustre_client_health::enabled: true` didn't scroll down enough when I copy pasted that. Fixed now